cli.js

```diff
@@ -26,11 +26,11 @@ var Promise = require('bluebird');
 var pkgConf = require('pkg-conf');
 var isCi = require('is-ci');
 var hasFlag = require('has-flag');
-var colors = require('./lib/colors');
+var colors = require('./lib/reporters/helpers/colors');
```

Comment:
Meh. I forgot all the places colors is used. Not sure about having it here now.

Maybe follow the same convention we use for test helpers?

```
lib/reporters/_colors
```

IDK.